lib/cn/cn_tree.c

```diff
@@ -116,8 +116,8 @@ tree_iter_next(struct cn_tree *tree, struct tree_iter *iter)
     struct cn_tree_node *visit = 0;
     struct cn_tree_node *prev = iter->prev;
     struct cn_tree_node *node = iter->next;
-    struct kvs_cparams * cp = tree->ct_cp;
-    u32                  child;
+    uint fanout = tree->ct_fanout;
```

Comment:
we go from a u32 fanout in kvs cparams to a u16 on the tree to an non-sized integer here. Should be consistent or cause pain in the future